topics/imaging/tutorials/multiplex-tissue-imaging-TMA/tutorial.md

```diff
@@ -0,0 +1,421 @@
+>    > 
+>    {: .comment}
+>
+{: .hands_on}
+
+
+# Interactive visualization of multiplex tissue images
+
+Visual analysis is an important part of multiplex tissue imaging workflows. Galaxy-MTI has several tools that make interactive visualization easy, and can be used at various stages of analysis. 
+
+## Converting UNetCoreograph images to OME-TIFF using the **Convert image** tool
+
+UNetCoreograph outputs each individual core image in `tiff` format. Interactive visualization tools, such as **Vitessce** and **Avivator** require the images to be in `OME-TIFF` format to be viewed. Galaxy-MTI includes a conversion tool that can accomodate this, along with many other useful conversion functions. 
+
+> ### {% icon hands_on %} Hands-on: Convert image
+>
+> 1. {% tool [Convert image](ip_convertimage) %} with the following parameters:
+>    - *"Output data type"*: `OME TIFF`
+>    - *"Extract series"*: `All series`
+>    - *"Extract timepoint"*: `All timepoints`
+>    - *"Extract channel"*: `All channels`
+>    - *"Extract z-slice"*: `All z-slices`
+>    - *"Extract range"*: `All images`
+>    - *"Extract crop"*: `Full image`
+>    - *"Tile image"*: `Tile image`
+>    - *"Pyramid image"*: `Generate Pyramid`
+>
+{: .hands_on}
+
+
+## **Rename OME-TIFF Channels**
+
+Some tools can cause the channel names in an OME-TIFF image to be lost. To fix this, or to change the channel names to whatever the user prefers, the **Rename OME-TIFF Channels** tool can be invoked using a markers file similar to the one used in previous steps. 
+
+> ### {% icon hands_on %} Hands-on: Rename channels
+>
+> 1. {% tool [Rename OME-TIFF Channels](toolshed.g2.bx.psu.edu/repos/watsocam/rename_tiff_channels/rename_tiff_channels/0.0.1.2) %} with the following parameters:
+>
+>    - {% icon param-file %} *"Input image in either tiff or OME-tiff format"*: File to be converted (or collection of files to run in batch)
+>    - *"Format of input image"*: `ome.tiff`
+>    - {% icon param-file %} *"Channel metadata CSV"*: Comma-separated markers file with marker_names in third column
+>
+{: .hands_on}
+
+
+## Initial visualization with **Avivator**
+
+For any `OME-TIFF` image in a Galaxy-MTI history, there will be an option to view the image using **Avivator**. This is a great way to perform an initial inspection of an image for QC purposes before continuing with downstream steps. The **Avivator** window can be launched by expanding the dataset information in the history panel and clicking the link (Figure 7.).
+
+![vivhistory](../../images/multiplex-tissue-imaging-TMA/ex2_avivatorHistory.png "The highlighted link automatically appears for any OME-TIFF image (left) and, when clicked, launches an Avivator window to explore the image (right).")
```

Comment:
Could you add a hands-on box for this step?

1. Open collection `name`
2. Expand a dataset
3. Click on "View at Aviator"